javascript/modules/client/test/integration/client.test.ts

```diff
@@ -532,7 +532,10 @@ describe("Client", () => {
         .updateMetadataRawAction(updateMetadataAction.data);
       const ipfsRegex =
         /^ipfs:\/\/(Qm([1-9A-HJ-NP-Za-km-z]{44,}|b[A-Za-z2-7]{58,}|B[A-Z2-7]{58,}|z[1-9A-HJ-NP-Za-km-z]{48,}|F[0-9A-F]{50,}))$/;
+
+      const expectedCid = await client.ipfs.add(JSON.stringify(params));
       expect(ipfsRegex.test(recoveredIpfsUri)).toBe(true);
+      expect(recoveredIpfsUri).toBe("ipfs://" + expectedCid);
```

Comment:
Testing the codebase with a parallel value that comes from the same underlying stack is a redundant test.
I would suggest to print this exact hash, hardcode it here, and let the test tell us if ever the same metadata produces a different value